### PR TITLE
Vets case study

### PR DESCRIPTION
**Pull request by dannychapman**
- Added new vets.gov case study
- revised home page case study bands styling
- added case study band styling for hover, active states based on touch / no-touch feature detection

### CODE REVIEW

**Comment by paulsmith** [comment]
Looking good, don't forget to add to `/case-studies/`

**Comment by dannychapman** [comment]
@paulsmith good catch, fixed!

**Comment by paulsmith** [comment]
LGTM :shipit:

File: pages/vets.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          <img class="feature" src="/assets/img/content/vets-full.png" alt="screenshot" />
+        </div>
+      </div>
+
+    <div class="write-up">
+        <div class="row">
+          <div class="small-12 medium-8 medium-push-4 columns main">
+            <h4>The Problem</h4>
+
+            <p>The department of Veterans Affairs needed to deliver a new beta website to quickly showcase how VA services could be conducted online.</p>
```
Comment (greggersh) [inline pages/vets.html]: Uppercase "department"

File: pages/vets.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+            <h4>The Solution</h4>
+
+            <p>Ad Hoc was engaged by the VA to help deliver the first version of vets.gov. Faced with an agressive eight-month timeline, the Ad Hoc team was engaged in all aspects of the project, including branding, user experience, visual design, front end coding, and technical infrastructure.</p>
```
Comment (greggersh) [inline pages/vets.html]: replace front-end coding w/ development

**Pull request closed without merging**